src/igz_controls/components/default-dropdown/default-dropdown.component.js

```diff
@@ -287,13 +295,28 @@
          */
         function onChangeTypingInput() {
             if (!lodash.isNil(ctrl.typedValue)) {
```

Comment:
I forgot to mention it, but when the input is empty then all options should be displayed.
I think for this you could change this to:
```js
if (lodash.isEmpty(ctrl.typedValue)) {
    ctrl.valuesArray = valuesArrayCopy;
} else {
    if (ctrl.autocomplete) { ... }
}
```
It's just an idea.
Please make sure that the drop-down menu is open after any change to the text input.